src/cascadia/WindowsTerminal/IslandWindow.h

```diff
@@ -32,10 +32,10 @@ class IslandWindow : public BaseWindow<IslandWindow>
     unsigned int _currentWidth;
     unsigned int _currentHeight;
 
-    HWND _interopWindowHandle;
-
     winrt::Windows::UI::Xaml::Hosting::DesktopWindowXamlSource _source;
 
+    HWND _interopWindowHandle;

```

Comment:
Why'd this move around?

Reply:
Was traces from an earlier attempt, and I ended up noticing that it wasn't a good idea to call CloseWindow on it later, causing oddball behavior...